packages/ember-metal/lib/meta_listeners.js

```diff
@@ -20,6 +22,24 @@ export var protoMethods = {
     if (!this._listeners) {
       this._listeners = [];
     }
+
+    runInDebug(() => {
+      // Assert that an observer is only used once.
+      let listeners = this._listeners;
+      let matchFound = false;
+
+      for (let i = listeners.length - 4; i >= 0; i -= 4) {
+        if (listeners[i+1] === target &&
```

Comment:
JSCS rules are failing for the lack of space between 'i' and '+' (lines below also).